docs/CONTRIBUTING.md

```diff
@@ -7,6 +7,11 @@ git clone git@github.com:kinvolk/lokomotive.git
 cd lokomotive
 ```
 
+## Requirements
+
+- Minimum Go version **1.4**
+- [golangci-lint](https://github.com/golangci/golangci-lint) installed locally
```

Comment:
This is not really a contributing requirement (`make` or `make build` does not require it). The CI will run linter anyway. But it's definitely a nice to have to run linter locally before submitting the PR.

Maybe we could add another section, for example "Testing and linting"?

Reply:
Yea testing section sounds great